search-app/src/cmr/search/api/concepts_search.clj

```diff
@@ -366,13 +368,28 @@
   ;; no errors, return 204
   {:status 204})
 
+(defn format-regex
+  "Adds the necessary string to each concept for a regex"
+  [concept] (str "(?" concept ")"))
+
+(def get-generics
+  "Retrieve the generics which were dynamically loaded"
+  (mapv keyword (mapv inf/plural(common-generic/latest-approved-document-types))))
+
+(def join-generic-concepts
+  "Combines the generic concepts to be a single string"
+  (clojure.string/join "|" (mapv format-regex get-generics)))
```

Comment:
other developers will complain that this is not defined above, clojure.string/join vs string/join